integration-tests/rust-wasi-tutorial/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rust-wasi-tutorial"
```

Comment:
```suggestion
name = "sample-file-cp"
```

Reply:
Hmm, I actually thought `rust-wasi-tutorial` was a good name especially since this is what the Yanga tutorial's example is called.

Reply:
Either way, if we're going to rename to `sample-file-cp` I'd like to propose renaming not only the name of the crate, but also the actual name of dir it resides in, plus the name of the testcase. How does that sound?

Reply:
I'd go for it. but maybe not in this PR ,because it needs syncing with yagna repo and possibly others (your private repo?).
It's will be self-explanatory at a first glance what does it do, and reader will be more engaged.

Reply:
Cool, let's do this in a subsequent PR this! :+1: